hydra-redis/hydra_graph.py

```diff
@@ -0,0 +1,80 @@
+import redis
+from redisgraph import Graph, Node
+import urllib.request as ur
+import json
+import re
+from hydrus.hydraspec import doc_maker
+from graphviz import Digraph
+from classes_objects import Classes
+from collections_endpoint import Collections as collect
+
+
+def final_file(url):
+    """ Open the given url and read and load the Json data."""
+    response = ur.urlopen(url)
+    return json.loads(response.read().decode('utf-8'))
+
+
+def get_apistructure(entrypoint_obj, url_node, api_doc):
+    """ It breaks the endpoint into two parts collection and classes"""
+    collection_endpoint = {}
+    classes_endpoint = {}
+    collection = 0
+    classes = 0
+    print("split entrypoint into 2 types of endpoints collection and classes")
+    for obj in url_node.properties:
+        if obj != "@id" and obj != "@type" and obj != "@context":
+            if obj in api_doc.collections.keys():
```

Comment:
I am not extracting the supportedProperty here I just extract the name and id of endpoints and divide into 2 types in this way:
`api_doc.entrypoint.get()` it will provide only and exact information about entrypoint that which endpoints are contained by entrypoint and with the use of `api_doc.collections.keys()` I can differentiate in class_endpoints and collection_endpoints.

Reply:
I know but it would be better if you use api_doc everywhere. Also, we've already separated Collection endpoints and Class endpoints using doc_writer why bother doing that again?

Reply:
Also, this will be a much cleaner approach.